2020-02-16 23:59:41,122 ERROR 73408 [-/::1/-/13ms POST /renovation/base/item] nodejs.TypeError: ctx.service.renovation.base.item is not a function
    at RenovationBaseController.item (/Users/lixin/project/egg-example/app/controller/renovation/base.js:16:52)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 73408
hostname: lixindeMacBook-Pro.local

2020-02-17 00:00:23,177 ERROR 73423 [-/::1/-/21ms POST /renovation/base/item] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'value' in 'field list'
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: INSERT INTO `renovation_base`(`id`, `typeName`, `name`, `type`, `value`) VALUES('705d4bc0-50d5-11ea-9f58-d50c3cde7a6c', '厨房', '厨房台盆', 'kitchen', '1131')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'value' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `renovation_base`(`id`, `typeName`, `name`, `type`, `value`) VALUES('705d4bc0-50d5-11ea-9f58-d50c3cde7a6c', '厨房', '厨房台盆', 'kitchen', '1131')"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 73423
hostname: lixindeMacBook-Pro.local

